e2e/tests/user/registration-ui.spec.ts

```diff
@@ -133,35 +135,47 @@ describe('User registration tests:', () => {
     const agreementPageButton = await createAccountPage.getNextButton();
     await agreementPageButton.click();
 
-    // Step 3: Enter user information. Should be on Create your account: Step 1 of 2 page
-    expect(await createAccountPage.waitForTextExists('Create your account')).toBeTruthy();
+    // Step 1 of 3: Enter Institution information
+    const nextButton = await createAccountPage.getNextButton();
+    expect(await nextButton.isCursorNotAllowed()).toEqual(true);
 
-    // the NEXT button on User Information page should be disabled until all required fields are filled
-    const userInforPageButton = await createAccountPage.getNextButton();
-    const cursor = await userInforPageButton.isCursorNotAllowed();
-    expect(cursor).toEqual(true);
+    const institutionSelect = new SelectComponent(page, 'Select your institution');
+    await institutionSelect.select(INSTITUTION_VALUE.VANDERBILT);
+
+    const emailAddress = await Textbox.forLabel(page, {textContains: FIELD_LABEL.INSTITUTION_EMAIL, ancestorNodeLevel: 2});
```

Comment:
Do we add negative test too? Like invalid email etc

Reply:
Good idea. I think that would be a new test in a new PR. purpose of this PR is mainly for fixing broken tests.